openml/testing.py

```diff
@@ -27,6 +29,38 @@ class TestBase(unittest.TestCase):
     Hopefully soon allows using a test server, not the production server.
     """
 
+    @pytest.fixture(scope="session", autouse=True)
+    def _cleanup_fixture(self):
+        """Cleans up files generated by Unit tests
+
+        This function is called at the beginning of the invocation of
+        TestBase (defined below), by each of class that inherits TestBase.
+        The 'yield' creates a checkpoint and breaks away to continue running
+        the unit tests of the sub class. When all the tests end, execution
+        resumes from the checkpoint.
+        """
+        # TODO: generalize this path better akin to static_cache_dir
```

Comment:
Yep, could you please work on this? Right now it appears that this unit test can only be executed from the main directory. You can have a look at how the static cache dir is obtained and use a similar logic here (except that you don't use `n_levels` but go back until you hit a directory called `openml` (which contains the package)).

Reply:
> go back until you hit a directory called `openml` (which contains the package))

so this would be to simulate `n_levels` wherein we go back till we are in the main directory again?

Reply:
Yes, that's the plan.